orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -231,93 +231,96 @@ def test_compare_pytorch_forward_call_positional_and_keyword_arguments(forward_s
     assert ortmodule_result == ortmodule_result_again
     assert pytorch_result == ortmodule_result
 
-def test_model_cuda():
+def test_torch_nn_module_cuda_method():
     original_device = 'cpu'
     to_device = 'cuda'
 
     N, D_in, H, D_out = 64, 784, 500, 10
     model = NeuralNetSinglePositionalArgument(D_in, H, D_out)
     model = ORTModule(model)
-    x = torch.randn(N, D_in, device=to_device)
     for _, parameter_value in model.named_parameters():
         assert parameter_value.device.type == original_device
 
+    x = torch.randn(N, D_in, device=to_device)
     model = model.cuda()
     model(x)
 
     for _, parameter_value in model.named_parameters():
         assert parameter_value.device.type == to_device
 
-def test_model_cpu():
+@pytest.mark.parametrize("set_gpu_on_original_module", [
+    True,
+    False
+    ])
+def test_torch_nn_module_cpu_api(set_gpu_on_original_module):
```

Comment:
minor comment here about keeping the test names consistent (use ```test_torch_nn_module_cpu_method``` instead of ```test_torch_nn_module_cpu_api```)? Similarly for other tests as well.